kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -27,18 +27,62 @@
           <div class="d-flex flex-column flex-grow-1">
-            <div class="d-flex flex-row">
-              <div class="d-flex flex-column flex-grow">
-                <span class="text-header-title my-auto mt-7 text-no-wrap">{{ $t('exoplatform.kudos.content.to') }} </span>
+            <div class="d-flex flex-row pt-5 align-center">
+              <span class="text-header-title">{{ $t('exoplatform.kudos.content.to') }}</span>
+              <div v-if="!isLinkedKudos" class="d-flex pr-2 pl-5">
+                <v-chip
+                  class="identitySuggesterItem me-4">
+                  <v-avatar left>
+                    <v-img :src="receiverAvatarUrl" />
+                  </v-avatar>
+                  <span class="text-truncate">
+                    {{ receiverFullName }}
+                  </span>
+                </v-chip>
               </div>
-              <div class="d-flex flex-column pr-2 pl-5 pt-3">
-                <div class="pt-3">
-                  <exo-user-avatar
-                    :identity="identity"
-                    bold-title
-                    link-style
-                    size="32" />
+              <div
+                v-else
+                class="d-flex flex-row pl-4 kudosReceiverAttendeeItem">
+                <div v-if="selectedReceiver" class="d-flex pr-2 pl-2 my-3 identitySuggesterInputStyle">
+                  <v-chip
+                    close
+                    @click:close="removeReceiver"
+                    class="identitySuggesterItem me-4 mt-2 mb-1">
+                    <v-avatar left>
+                      <v-img :src="receiverAvatarUrl" />
+                    </v-avatar>
+                    <span class="text-truncate">
+                      {{ receiverFullName }}
+                    </span>
+                  </v-chip>
                 </div>
+                <exo-identity-suggester
+                  v-else
+                  ref="invitedAttendeeAutoComplete"
+                  id="invitedAttendeeAutoComplete"
+                  v-model="selectedReceiver"
+                  :search-options="searchOptions"
+                  type-of-relations="member_of_space"
+                  name="inviteAttendee"
+                  include-users />
+              </div>
+            </div>
+            <div v-if="!isLinkedKudos">
+              <div class="d-flex flex-row pt-5">
+                <span class="text-header-title">{{ $t('exoplatform.kudos.choose.audience') }} </span>
+              </div>
+              <div class="d-flex flex-row pt-3">
+                <exo-identity-suggester
+                  ref="calendarOwnerSuggester"
+                  v-model="audience"
+                  :labels="spaceSuggesterLabels"
+                  :include-users="false"
+                  :width="220"
+                  name="calendarOwnerAutocomplete"
+                  class="user-suggester calendarOwnerAutocomplete"
+                  include-spaces
+                  only-redactor
```

Comment:
To check with PO if I can send kudos only to spaces where I 'm redactor (switch use case, it seems that it's indicated that even members)